.github/workflows/scripts/notify-consumers.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+import tomllib
+import json
+from urllib.request import Request, urlopen
+
+data = tomllib.load(open("pyproject.toml", "rb"))
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--event-type",
+    dest="event_type",
+    type=str,
+    default="MKDOCS_TECHDOCS_CORE::PUBLISHED_NEW_VERSION",
```

Comment:
Not sure why I chose this, it likely doesn't need to be **terribly** highly-specified, but figured I'd namespace it at least.